main.yml

```diff
@@ -0,0 +1,52 @@
+- hosts: localhost
+  gather_facts: no #optimization to gather facts only when required
+  vars_files:
+    - creds.yml
+  roles:
+    - "cloud_providers"
+ 
+- hosts: "tag_Env_Yugabyte_nodes*"
+  gather_facts: no 
+  pre_tasks:
+    - setup:
+  roles:
+    - yugabyte_common
+    - yugabyte_setup
+
+- hosts: tag_Env_Yugabyte_nodes_master
+  gather_facts:	no
+  pre_tasks:
+    - setup: # gather facts only when needed not every time
+  roles:
+    - role: cluster_control
+      vars:
+        file_config: master.conf
+        process: yb-master
+        basedir_yugabyte: "{{ master_dir }}"
```

Comment:
i am assuming this would run the control on the master nodes right?

Reply:
yes, basically on the nodes tagged follows:
- hosts: **tag_Env_Yugabyte_nodes_master**